Nodes/Decorators/RepeatUntilFail.cs

```diff
@@ -0,0 +1,29 @@
+﻿using UnityEngine;
+using System.Collections;
+
+namespace BehaviorTree
+{
+	public sealed class RepeatUntilFail : Decorator
+	{
+		public override TreeNode Init( Hashtable data )
+		{
+			return base.Init( data );
+		}
```

Comment:
why is this implementaion here and not just using the parent class implementation?